libp2p/protocols/connectivity/autonat/core.nim

```diff
@@ -58,6 +58,9 @@ type
     dial*: Option[AutonatDial]
     response*: Option[AutonatDialResponse]
 
+  NetworkReachability* {.pure.} = enum
+    NotReachable, Reachable, Unknown
```

Comment:
```suggestion
    Unknown, NotReachable, Reachable
```
Unknown should be the default value of this enum, this would also avoid having to set it explicitly to Unknown in the TCP constructor